src/CmsCoreLogBundle/objects/CronJobs/TPkgCmsCoreLogCleanupCronJob.class.php

```diff
@@ -36,23 +36,22 @@ protected function _ExecuteCron()
         $aChannelList = array();
         $oChannelList = TdbPkgCmsCoreLogChannelList::GetList();
         while ($oChannel = $oChannelList->Next()) {
-            $aChannelList[] = MySqlLegacySupport::getInstance()->real_escape_string($oChannel->fieldName);
-            $this->cleanChannel($oChannel->fieldName, $oChannel->fieldMaxLogAgeInSeconds, PKG_CMS_CORE_LOG_DEFAULT_MAX_AGE_IN_SECONDS_LEVEL_BELOW_WARNING);
+            $aChannelList[] = (string) MySqlLegacySupport::getInstance()->real_escape_string($oChannel->fieldName);
+            $this->cleanChannel($oChannel->fieldName, (int) $oChannel->fieldMaxLogAgeInSeconds, PKG_CMS_CORE_LOG_DEFAULT_MAX_AGE_IN_SECONDS_LEVEL_BELOW_WARNING);
         }
 
         $this->cleanOtherChannels($aChannelList, PKG_CMS_CORE_LOG_DEFAULT_MAX_AGE_IN_SECONDS, PKG_CMS_CORE_LOG_DEFAULT_MAX_AGE_IN_SECONDS_LEVEL_BELOW_WARNING);
     }
 
     /**
      * @param string $channelName
-     * @param string $iMaxAgeInSeconds
+     * @param int $iMaxAgeInSeconds
      * @param int $iMaxAgeInSecondsForLevelBelowNotice
      *
      * @return void
      */
     private function cleanChannel($channelName, $iMaxAgeInSeconds, $iMaxAgeInSecondsForLevelBelowNotice)
     {
-        $iMaxAgeInSeconds = intval($iMaxAgeInSeconds);
```

Comment:
In this instance, I changed the parameter type and decided to move the int-cast to the call a bit further up in the file